### DISCUSSION TITLE
Changing code for NBA

### DISCUSSION THREAD

**Discussion by weareallstardust** [riffnshred/nhl-led-scoreboard#291] [General]
Hi,

Thanks for your work on this. Looks fantastic. I am a newbie to Raspberry Pi's and wanted to change the code to show NBA scores for a selected team. I don't follow NHL, sorry. Where to I go to change that piece of information? I still want to keep the formatting about the same, just need it to show NBA scores with their respective team logos. 

Thanks!

**Comment by riffnshred** [comment]
Its not that simple sadly. Everything is programmed to organize and use the NHL data. The NBA and basketball data is different, so its not as simple as changing the source of data and the logos. 

There is an NBA scoreboard project that exist but it does not work the same way as mine. I would suggest starting there and work your way to make it look like mine.

https://github.com/bder222/NBA-scoreboard

**Reply by weareallstardust** [reply]
Oh thanks! That would work. I'll try that out and see if I can figure out how to add logos to it.
